MoviesGo/MoviesGo/DeteilModel/View/TabelViewCell/PosterPatchTableViewCell.swift

```diff
@@ -43,26 +38,21 @@ final class PosterPatchTableViewCell: UITableViewCell {
         ])
     }
 
+    func configCell(posterPath: String) {
+        serviceImage.updateImage(posterPath: posterPath) { [weak self] result in
+            guard let self = self else { return }
+            switch result {
+            case let .success(image):
+                self.posterImage.image = image
+            case let .failure(error):
+                self.posterImage.image = UIImage(named: "error")
```

Comment:
```suggestion
                self.posterImage.image = UIImage(named: "error")
```
"error" это хардкод ! всегда в константы

Reply:
убрал